packages/teams-js/src/public/liveShareHost.ts

```diff
@@ -261,8 +291,20 @@ export class LiveShareHost {
    * @beta
    */
   public static create(): LiveShareHost {
-    ensureInitialized(runtime, FrameContexts.meetingStage, FrameContexts.sidePanel);
+    ensureSupported();
 
     return new LiveShareHost();
   }
 }
+
+export function isSupported(): boolean {
```

Comment:
This function should have a comment above it. See dialog.url.isSupported for an example